speculos.py

```diff
@@ -88,7 +88,12 @@ def run_qemu(s1, s2, app_path, libraries=[], seed=DEFAULT_SEED, debug=False, tra
     # replace stdin with the socket
     os.dup2(s1.fileno(), sys.stdin.fileno())
 
-    seed = mnemonic.Mnemonic.to_seed(seed)
+    # handle both BIP39 mnemonics and hex seeds
+    if "hex:" in seed:
```

Comment:
I think something like `if seed.startswith("hex":)` might be better.

Reply:
Indeed !
Fixed.